decompiler/config/jak3/all-types.gc

```diff
@@ -5386,7 +5571,7 @@
   :method-count-assert 11
   :size-assert         #xa0
   :flag-assert         #xb000000a0
-  ;; field extra-sound-bank uses ~A with a signed load. 
+  ;; field extra-sound-bank uses ~A with a signed load.
```

Comment:
I stripped a bunch of trailing whitepsace, which is why the diff looks huge. Nothing past here should actually be changed.